server/app/main.py

```diff
@@ -199,17 +231,17 @@ async def start_transcription(
 
 # FIXME: this needs to be removed / put behind proper auth for security reasons
 @app.get("/tasks/list/")
```

Comment:
generally this needs another kind of auth, tho for now it seems to be ok

Reply:
yup, but i will leave the comment here to remind us to fix that in the future